joomla/joomla-cms
# [4.0] Fix default prefix for MvcFactory

Pull Request for Issue # .

### Summary of Changes
This is a propose fix for default $prefix parameter use in MvcFactory in Joomla 4. I know this is an ugly fix but could not find a better solution because we have to keep B/C with legacy MVC.

This needs code review and decision from Joomla 4 team.

### Testing Instructions
1. Install Joomla 4 staging
2. Apply patch
3. Play with com_content and make sure it works (backend)
4. Play with a none namespace component (banners or contacts) and make sure it works
